src/evergreen/metrics/buildmetrics.py

```diff
@@ -235,6 +331,16 @@ def as_dict(self, include_children=False):
             'pct_tasks_failed': self.pct_tasks_failed,
             'timed_out_count': self.timed_out_count,
             'system_failure_count': self.system_failure_count,
+            'total_display_tasks': self.total_display_tasks,
```

Comment:
There seems like there were already some fields missing from here - like pct system failure and pct timeout. I'm not sure why that is but didn't want to expand the scope of the ticket to fix it - so I basically only did as much as was necessary.